resources/terraform/testnet/digital-ocean/production.tfvars

```diff
@@ -5,6 +5,7 @@ bootstrap_droplet_image_id = 157362431
 node_droplet_size = "s-2vcpu-4gb"
 node_vm_count = 79
 node_droplet_image_id = 157362431
+private_node_vm_count = 1
```

Comment:
Oh, we may want to just set this to 0 for safety, just for now. Until we are really ready to introduce a home node into `PROD-02`. We will need to be certain about how that's going to work.

Reply:
Sounds good to me! Thanks for all the feedback!